### ISSUE TITLE
Enable Substitutable Text

### ISSUE THREAD

**Issue by reid-spencer** [reactific/riddl#196]

**Comment by JamesYopp** [comment]
So in the case where I have 2 parameters:

substitutions.yaml
`memberList: "A list of $thing to be $action an organization."`

mySpec.riddl
`command AddMembersToOrg is {???} briefly "${memberList:thing="member",action="added to"}"`

Is that what you were thinking?

**Comment by reid-spencer** [comment]
Yes, exactly. It's just naming the parameters instead of numbering them. That way order doesn't matter and it is more descriptive for the reader.

**Comment by reid-spencer** [comment]
Probably Hugo has a way of doing this or it can be conjured from Hugo/Go facilities.
